src/projectvariables.rs

```diff
@@ -0,0 +1,835 @@
+use liquid_core::Value;
```

Comment:
I would suggest renaming this file to have a `_` in the name, like `project_variables.rs`